server/odc-migrate/src/main/resources/migrate/common/V_4_2_3_5__modify_upload_file_config.sql

```diff
@@ -0,0 +1,19 @@
+--
+-- Modify upload file config
+--
+UPDATE
+  `config_system_configuration`
+SET
+  `value` = '100',
+  `description` = '异步任务最大上传文件数量，默认 100 个'
+WHERE
+  `key` = 'odc.flow.async.max-upload-file-count';
+
+UPDATE
+  `config_system_configuration`
+SET
+  `key` = 'odc.flow.async.max-upload-file-total-size',
+  `value` = '100',
+  `description` = '异步任务最大上传文件数量，默认 100 个'
+WHERE
+  `key` = 'odc.flow.async.max-upload-file-total-size}';
```

Comment:
Perhaps you need to **correct** the key name to `odc.flow.async.max-upload-file-total-size`, otherwise your business code can not load this configuration successfully.

Reply:
yes, the code is depracated